components/eventing-controller/README.md

```diff
@@ -14,82 +14,80 @@ This component contains controllers for various CustomResourceDefinitions relate
+| Environment Variable          | Description                                                                     |
+| ----------------------------- | ------------------------------------------------------------------------------- |
+| **For both**                  |                                                                                 |
+| `APP_LOG_FORMAT`              | The format of the Application logs.                                             |
+| `APP_LOG_LEVEL`               | The level of the Application logs.                                              |
+| `BACKEND_CR_NAMESPACE`        | The namespace of the Backend Resource (CR).                                     |
+| `BACKEND_CR_NAME`             | The name of the Backend Resource (CR).                                          |
+| `PUBLISHER_IMAGE`             | The image of the Event Publisher Proxy.                                         |
+| `PUBLISHER_IMAGE_PULL_POLICY` | The pull-policy of the Event Publisher Proxy.                                   |
+| `PUBLISHER_PORT_NUM`          | The port number of the Event Publisher Proxy itself.                            |
+| `PUBLISHER_METRICS_PORT_NUM`  | The port number of the Event Publisher Proxy metrics.                           |
+| `PUBLISHER_SERVICE_ACCOUNT`   | The service account of the Event Publisher Proxy.                               |
+| `PUBLISHER_REPLICAS`          | The number of replicas of the Event Publisher Proxy.                            |
+| `PUBLISHER_REQUESTS_CPU`      | The CPU requests of the Event Publisher Proxy.                                  |
+| `PUBLISHER_REQUESTS_MEMORY`   | The memory requests of the Event Publish Proxy.                                 |
+| `PUBLISHER_LIMITS_CPU`        | The CPU limits of the Event Publisher Proxy.                                    |
+| `PUBLISHER_LIMITS_MEMORY`     | The memory limits of the Event Publisher Proxy.                                 |
+| **For NATS**                  |                                                                                 |
+| `NATS_URL`                    | The URL for the NATS server.                                                    |
+| **For BEB**                   |                                                                                 |
+| `TOKEN_ENDPOINT`              | The Authentication Server Endpoint to provide Access Tokens.                    |
+| `WEBHOOK_CLIENT_ID`           | The Client ID used by webhooks to acquire Access Tokens from Kyma.              |
+| `WEBHOOK_CLIENT_SECRET`       | The Client Secret used by webhooks to acquire Access Tokens from Kyma.          |
+| `WEBHOOK_TOKEN_ENDPOINT`      | The Kyma public endpoint to provide Access Tokens.                              |
+| `DOMAIN`                      | The Kyma cluster public domain.                                                 |
```

Comment:
`WEBHOOK_ACTIVATION_TIMEOUT`, `EXEMPT_HANDSHAKE`, `QOS`, `CONTENT_MODE` are missing. You may find them [here](https://github.com/kyma-project/kyma/blob/main/components/eventing-controller/pkg/env/config.go#L36-L36).